docker/docker-compose.yml

```diff
@@ -181,3 +180,4 @@ volumes:
   redis-data:
   registry-data:
   postgresql-data:
+  celery-data:
```

Comment:
This isn't actually mounted to anything.  I assume it's supposed to be mounted up to `/var/lib/broker`.

Reply:
Resolved. Remnant from when I made the builder use the filesystem broker